UserPlugin-Creation.rst

```diff
@@ -39,12 +39,12 @@ named "trace" is active::
     from nuitka import Options
     from nuitka.plugins.PluginBase import UserPluginBase
 
-    class Usr_Plugin(UserPluginBase):
+    class MyPlugin(UserPluginBase):
 
-        plugin_name = __file__  # or __module__
+        plugin_name = __file__  # or __module__ or __name__
 
         def __init__(self):
-            # demo only: extract and display its list of options
+            # demo only: extract and display options list
```

Comment:
Hi. I think this comment should be "`extract and display the provided option with the plugin`" or something like that. Ref. https://github.com/Nuitka/Nuitka/blob/a9443b8eeb661ab70fa2ddb9c29c85192c78e391/nuitka/Options.py#L421